README.md

```diff
@@ -16,6 +16,7 @@ How to Add or Update a Job
 * Test your work
   * Push your feature branch to github
   * Request AWSInfrastructure@cru.org to turn on Jenkins Lab, if jenkins-lab.cru.org gives you a 503
+  * Merge your jenkins-job branch to jenkins-lab branch
```

Comment:
So, the cru-ansible setup on jenkins-lab is smart enough to run the jenkins-lab branch instead of master?

Reply:
@mattdrees that's the idea. Devs do not have access to the configure page.

Reply:
so, I see that https://jenkins-lab.cru.org/job/create-jenkins-jobs/configure has ` --extra-vars="jenkins_jobs_version=jenkins-lab"`, but that is not in this repo's simple-jobs.yaml. Did you manually add that? And if so, won't it get overwritten if we run the job?

Reply:
By the way, I'm not arguing against your proposed workflow; it is better for sure.
It's just that I want to make sure it's not broken.

Reply:
Thanks @mattdrees. It will need to be added there as well. I'll make the change. Appreciate the feedback! :)

Reply:
and yes it was added manually. I was not sure if it was configured with ansible or not. Good to see it's configured in this repo.